commit 8f485aececfd3adc895ce5f4a573c044937a4b1a
Author: Dexuan Cui <decui@microsoft.com>
Date:   Fri Feb 10 15:18:46 2017 -0600

    PCI: hv: Fix wslot_to_devfn() to fix warnings on device removal
    
    commit 60e2e2fbafdd1285ae1b4ad39ded41603e0c74d0 upstream.
    
    The devfn of 00:02.0 is 0x10.  devfn_to_wslot(0x10) == 0x2, and
    wslot_to_devfn(0x2) should be 0x10, while it's 0x2 in the current code.
    
    Due to this, hv_eject_device_work() -> pci_get_domain_bus_and_slot()
    returns NULL and pci_stop_and_remove_bus_device() is not called.
    
    Later when the real device driver's .remove() is invoked by
    hv_pci_remove() -> pci_stop_root_bus(), some warnings can be noticed
    because the VM has lost the access to the underlying device at that
    time.
    
    Signed-off-by: Jake Oshins <jakeo@microsoft.com>
    Signed-off-by: Dexuan Cui <decui@microsoft.com>
    Signed-off-by: Bjorn Helgaas <bhelgaas@google.com>
    Acked-by: Haiyang Zhang <haiyangz@microsoft.com>
    CC: K. Y. Srinivasan <kys@microsoft.com>
    CC: Stephen Hemminger <sthemmin@microsoft.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1290_8f485aececfd3adc895ce5f4a573c044937a4b1a.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1290_8f485aececfd3adc895ce5f4a573c044937a4b1a.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
